sympy/stats/rv.py

```diff
@@ -725,7 +728,7 @@ def probability(condition, given_condition=None, numsamples=None,
     given_condition = sympify(given_condition)
 
     if given_condition is not None and \
-            not isinstance(given_condition, (Relational, Boolean)):
+            not isinstance(given_condition, (Relational, Boolean, RandomSymbol)):
```

Comment:
This change would require a lot of testing.

Reply:
Agreed, though, is this change valid or the current code in master works fine?
I think for using `RandomSymbol` as `given_condition`, this should be there. Though, expressions like, `X+Y` will create a problem since, they are an object of `Add` class. 
I hope that I am not misinterpreting the requirement of the issue.